test/integration/066_test_selection_tests/test_selection_expansion.py

```diff
@@ -228,3 +232,32 @@ def test__postgres__exclude_data_test_tag(self):
         
         self.list_tests_and_assert(select, exclude, expected)
         self.run_tests_and_assert(select, exclude, expected)
+
+    @use_profile('postgres')
+    def test__postgres__model_a_greedy(self):
+        select = 'model_a'
+        exclude = None
+        greedy = True
+        expected = [
+            'cf_a_b', 'cf_a_src', 'just_a',
+            'relationships_model_a_fun__fun__ref_model_b_',
+            'relationships_model_a_fun__fun__source_my_src_my_tbl_',
+            'unique_model_a_fun'
+        ]
+            
+        self.list_tests_and_assert(select, exclude, expected, greedy)
+        self.run_tests_and_assert(select, exclude, expected, greedy)
```

Comment:
The `run_tests_and_assert` method takes other arguments (`schema` and `data`), so we need to specify the `greedy` arg by name, rather than just ordinal position:
```suggestion
        self.run_tests_and_assert(select, exclude, expected, greedy=greedy)
```

Reply:
doh